### ISSUE TITLE
Amazon.ae

### ISSUE THREAD

**Issue by 6--** [sushil-rgb/AmazonMe#14]
Hi,

Is it possible to support Amazon.ae?

Thanks a lot!

**Comment by sushil-rgb** [comment]
> Hi,
> 
> Is it possible to support Amazon.ae?
> 
> Thanks a lot!

I will look into it and will update you on the progress

**Comment by sushil-rgb** [comment]
Hey @6--  I have added support for .ae domains. Could you please check if it works for you?

**Comment by 6--** [comment]
> Hey @6-- I have added support for .ae domains. Could you please check if it works for you?

Thanks a lot! its working!

Just a question, is this normal `Error: 'NoneType' object has no attribute 'text'`?

base_url = "https://www.amazon.ae/s?k=razer&rh=n%3A12303875031&ref=nb_sb_noss"

```
, Integrated Silicone Dampening Foam, ABS Keycaps, AR layout, White
Retry 6 || Error: 'NoneType' object has no attribute 'text'
URL: https://www.amazon.ae/Razer-Rz04-02210100-R3M1-Electra-V2-Rz04-02210100-R3U1/dp/B075SV2Z29/ref=sr_1_20?dib=eyJ2IjoiMSJ9.w0hDG6QIlUlT3IFKQkr5a6SfUlbn7VN5XTN5dg-iioqU2NMqGfjBzyQGPQuerDM-DOQHRf_JZGJBl3Zwlg0v1EWhXUpIhetWB0WrR9ASXJcsIodXj8MXCR4UtwVQyYrHICusbozz1sl2sJJQ0-RGamJB3LiBm4X0rsPgd_wFGkYTtdXgeErw-joBa84iFmbg1jZ_R0fOimz0opVuMkIO0c8EJi-XDyEEtYARuryZsOQu1uAsQGGgVoVvZ140ARHuwyKU6V2n4WSv9GMSPX-OfNwGtE9fTLnSrrTSfNfc-X8.1GeUwJPVKobhMGyLMFPS2pS-BDHOn65uCJOs8sK10Go&dib_tag=se&keywords=razer&qid=1713285384&s=electronics&sr=1-20
EPOMAKER TH80 SE Gasket 75% Mechanical Keyboard, NKRO Hot Swappable RGB 2.4Ghz/ Bluetooth 5.0/ Wired Gaming Keyboard with Poron/EVA Foam, 4000mah Battery, Knob Control for E-sport/Windows/Mac
ASUS ROG Cetra True Wireless gaming headphones with low-latency wireless connection, ANC, up to 27-hour battery with wireless-charging case, IPX4 water resistance
Retry 7 || Error: 'NoneType' object has no attribute 'text'
URL: https://www.amazon.ae/Razer-Rz04-02210100-R3M1-Electra-V2-Rz04-02210100-R3U1/dp/B075SV2Z29/ref=sr_1_20?dib=eyJ2IjoiMSJ9.w0hDG6QIlUlT3IFKQkr5a6SfUlbn7VN5XTN5dg-iioqU2NMqGfjBzyQGPQuerDM-DOQHRf_JZGJBl3Zwlg0v1EWhXUpIhetWB0WrR9ASXJcsIodXj8MXCR4UtwVQyYrHICusbozz1sl2sJJQ0-RGamJB3LiBm4X0rsPgd_wFGkYTtdXgeErw-joBa84iFmbg1jZ_R0fOimz0opVuMkIO0c8EJi-XDyEEtYARuryZsOQu1uAsQGGgVoVvZ140ARHuwyKU6V2n4WSv9GMSPX-OfNwGtE9fTLnSrrTSfNfc-X8.1GeUwJPVKobhMGyLMFPS2pS-BDHOn65uCJOs8sK10Go&dib_tag=se&keywords=razer&qid=1713285384&s=electronics&sr=1-20
Razer Rz04-02210100-R3M1 Electra V2 - Analog Gaming Headset, Black, Rz04-02210100-R3U1
<pymongo.results.InsertManyResult object at 0x00000178B3C648E0>
```

**Comment by sushil-rgb** [comment]
> > Hey @6-- I have added support for .ae domains. Could you please check if it works for you?
> 
> Thanks a lot! its working!
> 
> Just a question, is this normal `Error: 'NoneType' object has no attribute 'text'`?
> 
> base_url = "https://www.amazon.ae/s?k=razer&rh=n%3A12303875031&ref=nb_sb_noss"
> 
> ```
> , Integrated Silicone Dampening Foam, ABS Keycaps, AR layout, White
> Retry 6 || Error: 'NoneType' object has no attribute 'text'
> URL: https://www.amazon.ae/Razer-Rz04-02210100-R3M1-Electra-V2-Rz04-02210100-R3U1/dp/B075SV2Z29/ref=sr_1_20?dib=eyJ2IjoiMSJ9.w0hDG6QIlUlT3IFKQkr5a6SfUlbn7VN5XTN5dg-iioqU2NMqGfjBzyQGPQuerDM-DOQHRf_JZGJBl3Zwlg0v1EWhXUpIhetWB0WrR9ASXJcsIodXj8MXCR4UtwVQyYrHICusbozz1sl2sJJQ0-RGamJB3LiBm4X0rsPgd_wFGkYTtdXgeErw-joBa84iFmbg1jZ_R0fOimz0opVuMkIO0c8EJi-XDyEEtYARuryZsOQu1uAsQGGgVoVvZ140ARHuwyKU6V2n4WSv9GMSPX-OfNwGtE9fTLnSrrTSfNfc-X8.1GeUwJPVKobhMGyLMFPS2pS-BDHOn65uCJOs8sK10Go&dib_tag=se&keywords=razer&qid=1713285384&s=electronics&sr=1-20
> EPOMAKER TH80 SE Gasket 75% Mechanical Keyboard, NKRO Hot Swappable RGB 2.4Ghz/ Bluetooth 5.0/ Wired Gaming Keyboard with Poron/EVA Foam, 4000mah Battery, Knob Control for E-sport/Windows/Mac
> ASUS ROG Cetra True Wireless gaming headphones with low-latency wireless connection, ANC, up to 27-hour battery with wireless-charging case, IPX4 water resistance
> Retry 7 || Error: 'NoneType' object has no attribute 'text'
> URL: https://www.amazon.ae/Razer-Rz04-02210100-R3M1-Electra-V2-Rz04-02210100-R3U1/dp/B075SV2Z29/ref=sr_1_20?dib=eyJ2IjoiMSJ9.w0hDG6QIlUlT3IFKQkr5a6SfUlbn7VN5XTN5dg-iioqU2NMqGfjBzyQGPQuerDM-DOQHRf_JZGJBl3Zwlg0v1EWhXUpIhetWB0WrR9ASXJcsIodXj8MXCR4UtwVQyYrHICusbozz1sl2sJJQ0-RGamJB3LiBm4X0rsPgd_wFGkYTtdXgeErw-joBa84iFmbg1jZ_R0fOimz0opVuMkIO0c8EJi-XDyEEtYARuryZsOQu1uAsQGGgVoVvZ140ARHuwyKU6V2n4WSv9GMSPX-OfNwGtE9fTLnSrrTSfNfc-X8.1GeUwJPVKobhMGyLMFPS2pS-BDHOn65uCJOs8sK10Go&dib_tag=se&keywords=razer&qid=1713285384&s=electronics&sr=1-20
> Razer Rz04-02210100-R3M1 Electra V2 - Analog Gaming Headset, Black, Rz04-02210100-R3U1
> <pymongo.results.InsertManyResult object at 0x00000178B3C648E0>
> ```

Yes it's normal, sometimes the Page doesn't render the content properly so the script will retry again to scrape the content
